qemu/tests/cfg/luks_image_over_qsd.cfg

```diff
@@ -0,0 +1,37 @@
+- luks_image_over_qsd:
+    only Linux
+    type = luks_image_over_qsd
+    required_qemu = [6.0, )
+    sleep_time = 5
+    start_vm = no
+    images += " nbd1"
+# Declare images attributes
```

Comment:
what about aligning the comment lines with the other lines

Reply:
Agree with Zhenchao, suggest also update line 13, 15, 18, 19, 21, 23, 25 and 28. Thanks.